cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/ProvisioningService.java

```diff
@@ -297,7 +297,7 @@ Runnable deprovision(ProgramRunId programRunId, DatasetContext datasetContext,
     ProvisioningTaskInfo existing =
       provisionerDataset.getTaskInfo(new ProvisioningTaskKey(programRunId, ProvisioningOp.Type.PROVISION));
     if (existing == null) {
-      runWithProgramLogging(programRunId, existing.getProgramOptions().getArguments().asMap(),
+      runWithProgramLogging(programRunId, Collections.emptyMap(),
```

Comment:
Why do we need this change?

Reply:
this was part of another fix that just got merged that was preventing me from running the sandbox. Basically, getting here would always result in a NPE, since existing is null